sigle/src/modules/layout/components/DashboardLayout.tsx

```diff
@@ -54,11 +60,62 @@ const NavItem = ({ children, ...props }: NavItemProps) => {
   );
 };
 
+interface AccordionNavItemProps {
+  href?: string;
+  children: React.ReactNode;
+}
+
```

Comment:
created a seperate component from `NavItem` as there were a few differences between the nav items in the sidebar and the ones in the accordion

Reply:
This approach makes much more sense, and also solves the warning in the console, thanks! 

Following the design, the items still need background colour when matching that particular path. Do you think it's better to add this functionality to them individually or create a wrapping component (similar to above) where that logic can be added? Hope that makes sense 😅

Reply:
I think using variants is the way for this :)